arrow-core/src/main/kotlin/arrow/core/extensions/const/apply/ConstApply.kt

```diff
@@ -100,8 +100,8 @@ fun <A, B, Z> Kind<Kind<ForConst, A>, A>.map2Eval(
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-    "map(MA, arg0, arg1, arg2)",
-    "arrow.core.Const.map"
+    "mapN(MA, arg0, arg1, arg2)",
+    "arrow.core.mapN"
```

Comment:
```suggestion
    "Const.mapN(MA, arg0, arg1, arg2)",
    "arrow.core.Const"
```

Idem for others in this file.